### PR TITLE
Enable #![deny(missing_docs)] on the SGX shim

### PR DESCRIPTION
**Pull request by npmccallum**

### CODE REVIEW

File: enarx-keep-sgx-shim/src/handler.rs [hunk 1]
```diff
@@ -207,7 +207,7 @@ impl<'a> Handler<'a> {
         let code = code
             .into()
             .map(|x| x.into())
-            .unwrap_or(self.aex.gpr.rdi.raw());
+            .unwrap_or_else(|| self.aex.gpr.rdi.raw());
```
Comment (lkatalin) [inline enarx-keep-sgx-shim/src/handler.rs:210]: Is this related to missing docs?

Comment (npmccallum) [reply enarx-keep-sgx-shim/src/handler.rs:210]: I'm not quite sure. I flipped the docs flag and the compiler started warning about this. Either way, I fixed it.
